source/runbooks/block-network-access-if-needed.html.md.erb

```diff
@@ -0,0 +1,169 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disabling network access if required
+last_reviewed_on: 2024-02-14
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+# Overview
+
+There may be occasions where there is a network attack on Modernisation Platform environments, or access insternally that is a cause for concern, that could require a way of blocking netwroking access/connectivity. This document lists the steps to follow.
+
+[NOTE]
+
+> All the steps below assume you can access the system through the **AWS console** and do not require changes to the terraform code. This is based on initial speed of access. Issues may also arise when amending the code, for example missing a step or code amendment issues, that could be caused following this route. Additionally the code would also need to be amended so the orignal steps are not lost so they can be re-instated once the issue is resolved. 
+There may be times when amending the code is quicker but this is not included here. Changes in done through this route will impact terraform code but that can be mitigated by manually reverting the steps followed to prevent access.
+
+# Check to see if there is an attack or unwanted access
```

Comment:
I don't think this is necessary. It should be stated at the start of the document that this document assumes the need to block network connectivity has been identified.